README.md

```diff
@@ -60,14 +60,12 @@ a link.
 
 Override the routing order like this:
 
-    Neos:
-      Flow:
-        http:
-          chain:
-            process:
-              chain:
-                redirect:
-                  position: 'before routing'
+Neos:
```

Comment:
The indentation is required for displaying it as code, or which would be even better to wrap it with ```yaml to produce correct highlighting.

Reply:
Good suggestion, thank you